egs/wsj/s5/local/e2e/run_end2end_phone.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+# Copyright 2017    Hossein Hadian
+
```

Comment:
Mention what this script is and also maybe mention similar scripts like _char.sh etc.